aten/src/ATen/core/function_schema.h

```diff
@@ -406,7 +406,7 @@ inline std::ostream& operator<<(std::ostream& out, const Argument& arg) {
 
   if (arg.default_value()) {
     out << "=";
-    if (type->kind() == c10::TypeKind::StringType) {
+    if (type->kind() == c10::TypeKind::StringType || unopt_type->kind() == c10::TypeKind::StringType) {
```

Comment:
The change itself looks good.

Just reminds me of current `Union` work by @ansley. Looks like this logic needs to be updated again after `Union` lands?